RealmSwift/Object.swift

```diff
@@ -212,11 +212,10 @@ extension Object: RealmCollectionValue {
      transactions it will be called at some point in the future after the write
      transaction is committed.
 
-     If no queue is given, notifications are delivered via the standard run
-     loop, and so can't be delivered while the run loop is blocked by other
-     activity. If a queue is given, notifications are delivered to that queue
-     instead. When notifications can't be delivered instantly, multiple
-     notifications may be coalesced into a single notification.
+     Notifications are delivered via the standard run loop, and so can't be
+     delivered while the run loop is blocked by other activity. When
+     notifications can't be delivered instantly, multiple notifications may be
+     coalesced into a single notification.
```

Comment:
The previous version of this wasn't great, but removing the mention of the queue isn't an improvement to it.